oaknational/Oak-Web-Application
# feat: terms and conditions for onboarding [LESQ-1001]

## Description

Music year: 1985

- adds text section with links to ts&cs
- slightly refactor school input so it doesn't start in an error state
- added additional tests and also noticed some act errors in onboarding page tests, not sure how I missed that but fixed them (there's some async actions in react-form that causes these so we have to await/find everything) 

## How to test

1. Go to https://deploy-preview-2665--oak-web-application.netlify.thenational.academy/onboarding (feature flag must be active for your distinct id)
3. You should see text below the form for oak's ts & cs and a link to contact us

## Screenshots

How it should now look:
<img width="512" alt="Screenshot 2024-08-05 at 11 57 52" src="https://github.com/user-attachments/assets/2ef1d1cc-66e5-4402-8b4a-cd2e6170fa91">

## Checklist

- [ ] school picker still works as expected and highlights error state / disables continue button
- [ ] links in text go to the correct places
- [ ] before merging: check feature flag is not enabled on prod except for specific user ids
